test/test_transformer.py

```diff
@@ -1505,6 +1505,18 @@ def test_4d_transform__inplace__numpy__int():
     assert tarr[0] == 2019
 
 
+def test_transform_single_point():
```

Comment:
Might not be a bad idea to parametrize `z` and `t` in this test for the cases that they are None (i.e. not provided).